### PR TITLE
Add feature task estimation

### PR DESCRIPTION
**Pull request by 4kh**
This is the draft documentation on task estimation, don't hesitate to add comments or content that may be missing
At the moment task estimation only contains project estimation and task estimation sub parts
Another part on SCRUM will be added later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,44 @@
...
+
+As seen before with project estimation, several steps are needed to correctly and efficiently estimate a task duration. Some of these are also applicable to the task scope.
+We will need to estimate not only the time we will need to accomplish the task but also many other things like human resources needed to handle the task, infrastructure cost, meetings needed to follow the task advancement, code reviews and many more.
+
+**Task estimation cons and pros**
+
+When talking about project or task estimation we obviously think about how the estimations will help the team to work efficiently and gain time on a project. But most of the time we don't take into account that taking too much time on estimations is also a lost of time in the sense that estimating could save us time thanks to a better organization but it's only worth if it's frequently done with short estimation and relatively quick decisions.
+
+[Read more about project and task estimation](https://medium.com/globalluxsoft/time-estimation-in-software-development-a4a495c8eb6c)
+
```
Comment (EtiennePASSOT) [inline README.md:42]: All is ok for me !

**Review by EtiennePASSOT — COMMENTED** [review]
all is ok

File: README.md [hunk 2]
```diff
@@ -2,6 +2,44 @@
...
+**What's a task ?**
+
+A task is defined by "a piece of work to be done or undertaken". So, in software progamming for example, a task could be a feature, a code review as well as writting documentation or resolving an issue.
+
+**How to estimate a task ?**
+
+As seen before with project estimation, several steps are needed to correctly and efficiently estimate a task duration. Some of these are also applicable to the task scope.
+We will need to estimate not only the time we will need to accomplish the task but also many other things like human resources needed to handle the task, infrastructure cost, meetings needed to follow the task advancement, code reviews and many more.
+
```
Comment (Arakhaa) [inline README.md:36]: I like this last point

File: README.md (hunk 1 above)
Comment (Arakhaa) [reply README.md:42]: Yeah, it looks great

**Review by EtiennePASSOT — APPROVED** [review]

**Comment by valotg** [comment]
All right for me

**Review by Arakhaa — APPROVED** [review]

**Pull request merged**